src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -0,0 +1,324 @@
+ *                                      "multirate_triggers":[""],
+ *                                      "pl_kernel_instance_names":[""],
+ *                                      "pl_axi_lite_modes":[""]
+ *                                      }
+ *                            },
+ *                  "RTPs":{
+ *                          "rtp0":{
+ *                                  "port_id":"",
+ *                                  "alias_id":"",
+ *                                  "port_name":"",
+ *                                  "alias_name":"",
+ *                                  "graph_id":"",
+ *                                  "is_input":"",
+ *                                  "is_asynchronous":"",
+ *                                  "is_connected":"",
+ *                                  "element_type":"",
+ *                                  "number_of_bytes":"",
+ *                                  "is_PL_RTP":"",
+ *                                  "requires_lock":"",
+ *                                  "selector_column":"",
+ *                                  "selector_row":"",
+ *                                  "selector_address":"",
+ *                                  "selector_lock_id":"",
+ *                                  "ping_buffer_column":"",
+ *                                  "ping_buffer_row":"",
+ *                                  "ping_buffer_address":"",
+ *                                  "ping_buffer_lock_id":"",
+ *                                  "pong_buffer_column":"",
+ *                                  "pong_buffer_row":"",
+ *                                  "pong_buffer_address":"",
+ *                                  "pong_buffer_lock_id":"",
+ *                                  "pl_kernel_instance_name":"",
+ *                                  "pl_parameter_index":""
+ *                                  }, 
+ *                         },
+ *                  "GMIOs":{
+ *                           "gmio0":{
+ *                                     "id":"",
+ *                                     "name":"",
+ *                                     "logical_name":"",
+ *                                     "type":"",
+ *                                     "shim_column":"",
+ *                                     "channel_number":"",
+ *                                     "stream_id":"",
+ *                                     "burst_length_in_16byte":"",
+ *                                     "PL_port_name":"",
+ *                                     "PL_parameter_name":""
+ *                                     }
+ *                          }
+ *                  }
+ * }
+ */
+    for (auto& gr: _pt.get_child("aie_metadata.graphs")) {
+      boost::property_tree::ptree& ograph = gr.second;
+      boost::property_tree::ptree igraph;
+      boost::property_tree::ptree tile_array;
+      igraph.put("id",ograph.get<std::string>("id"));

```

Comment:
In the following, add space after `,` so the reader can keep breathing... :-)

Reply:
fixed